Privacy.md

```diff
@@ -0,0 +1,59 @@
+## Weather Widget Privacy Policy
+
+
+This is the developer's policy on how your personal information may be collected, used, disclosed and secured. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. 
+If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. Kindly note that this privacy policy is restricted only to Weather Widget software and the developer is not and will not be liable to be held in account for the privacy practices of any third-party tools or services that may be contained in the software.
+
+---
+
+## Information Collection And Use
+
+To function basically as intended, the developer doesn't collection any form of information personally when the software is utilized and the software doesn't require any permission either. It is a simple widget software. However, the software directly uses the OpenWeatherMap API. 
+
+
+---
+
+## Types of Data Collected
+
+### Personal Data
+
+Due to the integration of OpenWeatherMap API in the software personal data may be collected while the weather widget is in use. This data is in no way retrieved by the developer of Weather widget.
+
+### Usage Data
+
+This app does not record usage information, however, usage statistics are captured by OpenWeatherMap with respect to their [policy](https://openweather.co.uk/privacy-policy).
+
+### Use of Data
+
+With the integration of OpenWeatherMap API, the data collected is used to allow Weather widget 
+
+- Access current weather data for any location including over 200,000 cities.
+
+---
+
+## Security Of Data
+
+There are no security risks that may likely occur while using Weather Widget software. However, with the inclusion of OpenWeatherMap API, it is necessary to read how they(OpenWeatherMap) secure your data that may be collected while you utilize Weather Widget. The developers of Phone Saver assume no responsibility for data integrity issues caused by using this application.
```

Comment:
Can you change Phone Saver to Weather Widget?
And maybe add a space after they and before the '(OpenWeatherMap)'?